### PR TITLE
Panel styles

### PR DESCRIPTION
**Pull request by tomgatzgates**
This adds the ability to use panels as notification banners. 

Currently supports:
Info - blue (blue needs updating to be more Pacty. Suggestions welcome)
Success - green
Error - red

screen shot 2015-07-06 at 12 40 38

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "loggins",
-  "version": "1.2.0",
+  "version": "1.3.1",
```
Comment (tomgatzgates) [inline package.json]: Not sure what's going on here. I did;t manually change this

**Comment by iest** [comment]
Screenshots or it didn't happen

**Comment by tomgatzgates** [comment]
Added screenshots

**Comment by iest** [comment]
`--color-blue:         rgb(21,125,249);`

I used it to indicate focus... @scott-riley probably has other ideas considering his new account page stuff...

File: package.json (hunk 1 above)
Comment (iest) [inline package.json]: Can you change it just to `1.3.0`?

**Pull request closed without merging**